enrolment-service/enrol/core/migrations/0009_add_billing_contact_email_and_phone.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
```

Comment:
Autogenerated db migration to add the new fields, and fill the existing rows with empty strings.